lib/engine/game/g_1877_stockholm_tramways/game.rb

```diff
@@ -0,0 +1,236 @@
+# frozen_string_literal: true
+
+require_relative 'meta'
+require_relative '../base'
+require_relative 'entities'
+require_relative 'map'
+
+module Engine
+  module Game
+    module G1877StockholmTramways
+      class Game < Game::Base
+        include_meta(G1877StockholmTramways::Meta)
+        include Entities
+        include Map
+
+        register_colors(black: '#000000')
+
+        CURRENCY_FORMAT_STR = '%dkr'
```

Comment:
Compare currency format string in 18SJ.

Reply:
I did check but I preferred to be consistent with my own rules.